lib/init.coffee

```diff
@@ -16,24 +16,26 @@ module.exports =
   V:{}
   Status:{}
   activate:->
-    @V.MP = require('atom-message-panel')
-    @V.H = window.Atom_HACK_H = require('./h')(this)
-    @V.TE = require('./typechecker-error')(this);
-    @V.TTV = require('./tooltip-view')(this);
+    setTimeout =>
+      @V.MP = require('atom-message-panel')
+      @V.H = window.Atom_HACK_H = require('./h')(this)
+      @V.TE = require('./typechecker-error')(this);
+      @V.TTV = require('./tooltip-view')(this);
 
-    require('./cmenu')(this).initialize();
-    @V.MPI = new @V.MP.MessagePanelView title: "Hack TypeChecker"
+      require('./cmenu')(this).initialize();
+      @V.MPI = new @V.MP.MessagePanelView title: "Hack TypeChecker"
 
-    @Status.TypeChecker = false
-    @V.H.readConfig().then =>
-      setTimeout =>
-        @V.TC = require('./typechecker')(this);
-        atom.config.observe 'Atom-Hack.enableTypeChecking',(status)=>
-          if status
-            @V.TC.activate()
-          else
-            @V.TC.deactivate()
-      ,500
+      @Status.TypeChecker = false
+      @V.H.readConfig().then =>
+        setTimeout =>
```

Comment:
Lets just improve this timeout instead of wrapping the whole part in setTimeout, 'cause the autocomplete.coffee is loaded __completely separately__ and we need to prepare a global variable for it or otherwise it'll throw some errors.